src/lib/components/list-editor/list-editor.svelte

```diff
@@ -30,6 +32,7 @@
   export let allowDripLists = true;
   export let allowProjects = true;
   export let allowAddresses = true;
+  export let allowOrcids = true;
```

Comment:
maybe i'm missing something, but i don't see this being set to `false` in list editor usages? might be best to set to false by default for now to ensure no list-editors anywhere suddenly start accepting orcids, especially on networks where it's not yet enabled